src/Microsoft.AspNetCore.Mvc.Abstractions/Routing/AttributeRouteInfo.cs

```diff
@@ -26,5 +26,15 @@ public class AttributeRouteInfo
         /// route by provided route data.
         /// </summary>
         public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value that determines if this model participates in link generation.
```

Comment:
determines if **the route entry associated with** this model participates in link generation - and similar for the other comment